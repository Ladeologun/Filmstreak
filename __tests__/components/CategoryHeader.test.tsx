import renderer from 'react-test-renderer';
import {test,expect} from '@jest/globals';
import { CategoryHeader } from "~components";


test('renders correctly', () => {
    const tree = renderer.create(<CategoryHeader title='filmstreak' />).toJSON();
    expect(tree).toMatchSnapshot();
});
