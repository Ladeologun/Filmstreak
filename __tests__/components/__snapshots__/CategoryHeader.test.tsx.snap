// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flexDirection": "row",
      "justifyContent": "space-between",
      "marginBottom": 20,
      "paddingBottom": 5,
      "paddingTop": 15,
    }
  }
>
  <Text
    style={
      {
        "borderBottomColor": "green",
        "color": "#ffffff",
        "fontFamily": "PermanentMarker-Regular",
        "fontSize": 19,
        "fontWeight": "800",
        "textDecorationColor": "gray",
        "textShadowColor": "gray",
        "textShadowOffset": {
          "height": 0.5,
          "width": 0.5,
        },
      }
    }
  >
    filmstreak
  </Text>
</View>
`;
